src/storage/hummock_sdk/src/version.rs

```diff
@@ -190,3 +436,639 @@ impl HummockVersionDelta {
         }
     }
 }
+
+#[derive(Debug, PartialEq, Clone, Default, Serialize)]
+pub struct SstableInfo {
+    pub object_id: u64,
+    pub sst_id: u64,
+    pub key_range: Option<KeyRange>,
```

Comment:
I think we can use `FullKeyRange` instead rather than the `KeyRange`. The field `is_right_inclusive` is very implicit.